src/applications/mhv/medical-records/components/CareSummaries/AdmissionAndDischargeDetails.jsx

```diff
@@ -100,6 +104,27 @@ const AdmissionAndDischargeDetails = props => {
     );
   };
 
+  const generateCareNotesTxt = () => {
+    const content = `\n
+${record.name}\n
+Date: ${formatDateLong(record.date)}\n
+_____________________________________________________\n\n
+Details about this test\n
```

Comment:
The notes aren't tests, so we probably want different text here. See UCD for correct verbiage. Same for Progress Notes.

Reply:
I remember UCD wanting me to make it match the PDF. I just imitated the sketch.